app/models/notification_phone_configuration.rb

```diff
@@ -0,0 +1,27 @@
+class NotificationPhoneConfiguration < ApplicationRecord
+  include EncryptableAttribute
+
+  belongs_to :in_person_enrollment, inverse_of: :notification_phone_configuration
+
+  encrypted_attribute(name: :phone)
+
+  def formatted_phone
+    Phonelib.parse(phone).international
+  end
+
+  def masked_phone
+    return '' if phone.blank?
+
+    formatted = Phonelib.parse(phone).national
+    formatted[0..-5].gsub(/\d/, '*') + formatted[-4..-1]
+  end
+
+  def erase_phone_number_and_mark_notification_sent
+    self.notification_sent_at = Time.zone.now
+    self.phone = nil
+  end
```

Comment:
I'm surprised by this erase behavior, I skimmed the JIRA and associated tech spec doc, can you link me to more information on it?

I feel like we could go a few directions:
- If we're adding a new table for explicit phone notifications, that seems like a reasonable thing to keep around
- If we really do want to make sure these are one-time use, should we just delete the row? Or we could do something like we have with `deleted_users` where we create essentially a "tombstone" table that keeps a reference to things?

Reply:
I don't think there is any more info on it, the idea is just to delete pii once we no longer need it. This was the suggested implementation for the ticket but I could also see us just deleting the `notification_phone_configuraiton` record once send a notification and moving the `notification_sent_at` column to the `in_person_enrollments` table. So basically the second of your two directions

Reply:
Yeah makes sense! I think having a phone number row with a null value is confusing, especially if we have zero or one of these attached to a user, so if there was ever a second time a user goes through the flow it gets confusing

Reply:
I would be fine with moving `notification_sent_at` to the enrollment and deleting the `notification_phone_configuration`

Reply:
Done in 08db9e59798f313831ff2f6eba07937877a42e1b. I'm going to defer the `notification_sent_at` column to a future ticket since it won't actually be used as part of this ticket